content/blog/2-dangers-from-foreign-influence/index.md

```diff
@@ -99,7 +99,7 @@ Long Farewell, to All my Greatness."
 
 ---
 
-Footnote 1: The lack of diversity in these lines grates on the modern ears. We recognize the great benefit
+<a name="footnote1">Footnote 1</a>: The lack of diversity in these lines grates on the modern ears. We recognize the great benefit
```

Comment:
I think what you really want is to surround this whole thing with `<div id="footenote1"> ... </div>`

I don't think putting a name on an anchor tag will work. I've never seen that, anyway.

Reply:
It's been a long time since I've done much in HTML - I just looked up how to make footnotes in markdown and copied and pasted. :)